clang/test/dpct/helper_files_ref/include/device.hpp

```diff
@@ -598,12 +627,25 @@ class dev_mgr {
   int _cpu_device = -1;
 };
 
```

Comment:
/// Util function to get the default queue of current selected device depends on the USM config.
/// Return the default out-of-ordered queue when
/// USM-none is enabled, otherwise return the default in-ordered queue